src/sentry/tasks/auto_archive_issues.py

```diff
@@ -45,7 +45,7 @@ def run_auto_archive() -> None:
     for organization in RangeQuerySetWrapper(
         Organization.objects.filter(status=OrganizationStatus.ACTIVE)
     ):
-        if features.has("organizations:escalating-issues", organization):
+        if features.has("organizations:escalating-issues-v2", organization):
```

Comment:
Won't doing this disable this feature for existing projects that have it enabled? Or is this jus tnot running currently?